packages/mdc-layout-grid/README.md

```diff
@@ -70,24 +68,35 @@ Behavior for grids containing direct children without the `mdc-layout-grid__cell
 
 ### Margins and gutters
 
-The default size for both the margins and gutters in MDC layout grid is 16px.
+Layout grids set default margins and gutters to 24px on desktop, 16px on tablet and phone.
 
-You can change the margins and gutters for a grid using the `--mdc-layout-grid-margin` and `--mdc-layout-grid-gutter`
-custom properties, respectively. This requires support for CSS custom properties on the end-user's browser.
+The Material Design spec recommends 8px, 16px, 24px or 40px as the sizes to choose from, we set those as choices in our demo catalog. However, MDC layout grid doesn't impose any restrictions.
 
-The Material Design spec recommends 8px, 16px, 24px or 40px as the sizes to choose from, but MDC layout grid doesn't
-impose any restrictions.
+> Note: Due to the implementation of MDC layout grid, it's not possible to use a margin smaller than half of the size
+of the gutter, in most browsers. As support for [CSS Grid](https://www.w3.org/TR/css-grid-1/) lands in browsers, this
+limitation will go away, as MDC layout grid is progressively enhanced to support it.
+
+
+#### CSS custom properties
+
+You can change the margins and gutters for a grid using the `--mdc-layout-grid-margin-#{$device}` and `--mdc-layout-grid-gutter-#{$device}` custom properties, respectively. This requires support for CSS custom properties on the end-user's browser.
 
 ```css
 .my-grid {
-  --mdc-layout-grid-margin: 40px;
-  --mdc-layout-grid-gutter: 16px;
+  --mdc-layout-grid-margin-desktop: 40px;
+  --mdc-layout-grid-gutter-tablet: 16px;
```

Comment:
nit: perhaps we should also show `--mdc-layout-grid-gutter-phone` being used here?